airflow/providers/amazon/aws/operators/datasync.py

```diff
@@ -335,9 +335,10 @@ def _execute_datasync_task(self):
         # Log some meaningful statuses
         level = logging.ERROR if not result else logging.INFO
         self.log.log(level, 'Status=%s', task_execution_description['Status'])
-        for k, v in task_execution_description['Result'].items():
-            if 'Status' in k or 'Error' in k:
-                self.log.log(level, '%s=%s', k, v)
+        if 'Result' in task_execution_description:
```

Comment:
you might want to unpin the moto version too for the tests

Reply:
That is a great point, thank you.
I tested with moto 1.3.16 on Windows just using pytest directly, and forgot to unpin :)
Unpinned now, let's see what the build does.